### ISSUE TITLE
Prop `style` did not match

### ISSUE THREAD

**Issue by simoncarbajal** [rsuite/rsuite-table#151]
I'm having something like this:
```jsx
import { Table } from 'rsuite'
<Table data={data} loading={loading} hover height={400} autoHeight>
    <Column flexGrow={2}>
      <HeaderCell>...</HeaderCell>
      <Cell dataKey='key' />
    </Column>
    ...
</Table>
```
And I'm rendering the page in the server, which causes this stack trace (in the browser):
```
Warning: Prop `style` did not match. Server: "left:0px;top:0px" Client: "transform:translate3d(0px,0px,0);backface-visibility:hidden"
    in div (created by Context.Consumer)
    in CellGroup (created by Table)
    in div (created by Context.Consumer)
    in Row (created by Table)
    in div (created by Table)
    in div (created by Table)
    in Table (created by defaultProps(Table))
    in defaultProps(Table) (created by Context.Consumer)
    in withLocale(defaultProps(Table)) (created by Results)
    in div (created by Context.Consumer)
    in StyledComponent (created by styles__ContentContainer)
    in styles__ContentContainer (created by Results)
    in Results (created by withState(Results))
    in withState(Results) (created by Connect(withState(Results)))
    in Connect(withState(Results)) (created by MainPage)
    in div (created by Context.Consumer)
    in StyledComponent (created by styles__Page)
    in styles__Page (created by MainPage)
    in MainPage (created by withState(MainPage))
    in withState(MainPage) (created by Connect(withState(MainPage)))
    in Connect(withState(MainPage)) (created by MyApp)
    in Provider (created by MyApp)
    in div (created by MyApp)
    in MyApp (created by withRedux(MyApp))
    in withRedux(MyApp) (created by withRouter(withRedux(MyApp)))
    in withRouter(withRedux(MyApp))
    in Container (created by AppContainer)
    in AppContainer
```
Any clue?

**Comment by simonguo** [comment]
This should be a warning in server-side rendering. rsuite-table rendering depends on the browser environment. I recommend that you render rsuite-table rendering only in the browser.

**Comment by simonguo** [comment]
Fixed in rsuite-table@5.3.3
